src/components/ChatRoom.tsx

```diff
@@ -103,9 +105,33 @@ function Editor({ roomId, message, onClose, ...rest }: {
     message ? message.markdown : "");
   const [saving, setSaving] = useState<boolean>(false);
   const utils = trpcNext.useContext();
-  
-  const insertSnippet = (snippet: string) => {
-    setMarkdown(prev => prev + snippet);
+  const direction: "row" | "column" | undefined = useBreakpointValue({
```

Comment:
Why not at line 160, simply do `direction={{ base: "column", [sidebarBreakpoint]: "row" }}`?